code/datums/cookingrecipes.dm

```diff
@@ -1073,6 +1073,27 @@
 	cookbonus = 15
 	output = /obj/item/reagent_containers/food/snacks/pie/ass
 
+/datum/cookingrecipe/pot_pie
+	item1 = /obj/item/reagent_containers/food/snacks/ingredient/dough_s
```

Comment:
Sweet dough??

Reply:
Wouldn't like some butter make more sense? Baking with weed you'd wanna get the fat contents in the dough real high so the cannaboids bind to the batter and activate them for enjoyment.

Also butter at least feels to me like it is barely touched upon.
-Shrug; merely a thought.-

Reply:
Changed it to just be normal dough.

Reply:
Okay; I actually went and tested the new changes with the normal dough in the recipe and the problem is it reads the ingredient combination and produces burgers instead, and I don't think there's a way to resolve this for something like differentiating between the fish pie and a fish burger since they use the same ingredients and cook time range/method. If it's cool with you, I'd like to leave the dough as sweet dough versus trying to order the pies above custom burgers.

Reply:
sure ok